app/reducers/views/channel.js

```diff
@@ -23,14 +23,22 @@ function drafts(state = {}, action) {
     case ViewTypes.POST_DRAFT_CHANGED: {
         return {
             ...state,
-            [action.channelId]: Object.assign({}, state[action.channelId], {draft: action.postDraft})
+            [action.channelId]: Object.assign({}, state[action.channelId], {draft: action.draft})
         };
     }
+    case ViewTypes.POST_DRAFT_SELECTION_CHANGED:
+        return {
```

Comment:
question: doesn't  the position changes when you type? wondering if we can simplify this and dispatch just one action type instead of two

Reply:
It does, but this also captures if the user moves the cursor position without typing, like to insert before a certain word if they have already typed it.